data/lib/core/constants.lua

```diff
@@ -30,3 +32,13 @@ SCREENSHOT_TYPE_SKILLUP = 12
 
 SCREENSHOT_TYPE_FIRST = SCREENSHOT_TYPE_ACHIEVEMENT
 SCREENSHOT_TYPE_LAST = SCREENSHOT_TYPE_SKILLUP
+
+BLESS_TYPE_ADVENTURER = -1 -- not supported
```

Comment:
you have a negative value there

overall, I believe that this should be an enum in `const.cpp` pushed to Lua through `registerEnum` in `luascript.cpp`

Reply:
The numbering is actually incorrect, it should be 1 and not -1. However, I see no reason to move to the engine as it is not being used anywhere.